hardware/raspberrypi/bcm2837/README.md

```diff
@@ -4,4 +4,8 @@ This is the Broadcom chip used in the Raspberry Pi 3, and in later models of the
 
 The ARM cores run at 1.2GHz, making the device about 50% faster than the Raspberry Pi 2. The VideoCore IV runs at 400MHz.
 
+Please refer to the following BCM2836 document for details on the ARM peripherals specification which is unchanged.
```

Comment:
"...which also applies to the BCM2837." is less ambiguous.

Reply:
Done